From b6d74c7e314d4c446bff036627af6684f197ad7b Mon Sep 17 00:00:00 2001
From: mbrinkma <mbrinkma@2eb42a87-3827-0410-b767-dc15055982e4>
Date: Fri, 26 Nov 2021 08:10:51 +0000
Subject: [PATCH 435/462] MANTIS 0061953: create end2end-test-cases checking
 ADA-compliance - build new accessibility-device-simulator-service for
 enabling the above tests

git-svn-id: svn://localhost/SelfServiceBuild/trunk@826 2eb42a87-3827-0410-b767-dc15055982e4
---
 SelfServiceBuild/Build/scripts/gradle-build-all.bat | 1 +
 1 file changed, 1 insertion(+)

diff --git a/SelfServiceBuild/Build/scripts/gradle-build-all.bat b/SelfServiceBuild/Build/scripts/gradle-build-all.bat
index 96729a6..aadb7dd 100644
--- a/SelfServiceBuild/Build/scripts/gradle-build-all.bat
+++ b/SelfServiceBuild/Build/scripts/gradle-build-all.bat
@@ -26,6 +26,7 @@ call :callGradle %BASE_DIR%\DeviceSimulation\illumination-device-sim %BUILD_ARGS
 rem call :callGradle %BASE_DIR%\DeviceSimulation\device-reg-test %BUILD_ARGS%
 call :callGradle %BASE_DIR%\BaggageCommunicationLib %BUILD_ARGS%
 call :callGradle %BASE_DIR%\AirCanadaBIWIS\BiwisHandler %BUILD_ARGS%
+call :callGradle %BASE_DIR%\DeviceSimulation\Accessibility\service %BUILD_ARGS%
 
 goto :eof
 
-- 
2.41.0.windows.1

